PA_NSSDB_DIR = "/etc/ipa/nssdb"
     IPA_NSSDB_PWDFILE_TXT = "/etc/ipa/nssdb/pwdfile.txt"
     COMMON_KRB5_CONF_DIR = "/etc/krb5.conf.d/"
-    KRB5_CONF = "/etc/krb5.conf"
-    KRB5_KEYTAB = "/etc/krb5.keytab"
+    KRB5_CONF = "/data/etc/krb5.conf"
+    KRB5_KEYTAB = "/data/etc/krb5.keytab"
     LDAP_CONF = "/etc/ldap.conf"
     LIBNSS_LDAP_CONF = "/etc/libnss-ldap.conf"
     NAMED_CONF = "/etc/named.conf"
     NAMED_VAR_DIR = "/var/named"
-    NAMED_KEYTAB = "/etc/named.keytab"
+    NAMED_KEYTAB = "/data/etc/named.keytab"
     NAMED_RFC1912_ZONES = "/etc/named.rfc1912.zones"
     NAMED_ROOT_KEY = "/etc/named.root.key"
     NAMED_BINDKEYS_FILE = "/etc/named.iscdlv.key"
     NAMED_MANAGED_KEYS_DIR = "/var/named/dynamic"
     NSLCD_CONF = "/etc/nslcd.conf"
     NSS_LDAP_CONF = "/etc/nss_ldap.conf"
-    NSSWITCH_CONF = "/etc/nsswitch.conf"
+    NSSWITCH_CONF = "/data/etc/nsswitch.conf"
     NTP_CONF = "/etc/ntp.conf"
     NTP_STEP_TICKERS = "/etc/ntp/step-tickers"
     ETC_OPENDNSSEC_DIR = "/etc/opendnssec"
--- /etc/pki/default.cfg	2018-08-17 15:37:39.000000000 +0000
+++ /etc/pki/default.cfg	2018-09-15 10:20:58.765930755 +0000
@@ -196,7 +196,7 @@
 pki_source_subsystem_path=/usr/share/pki/%(pki_subsystem_type)s
 pki_path=%(pki_root_prefix)s/var/lib/pki
 pki_log_path=%(pki_root_prefix)s/var/log/pki
-pki_configuration_path=%(pki_root_prefix)s/etc/pki
+pki_configuration_path=/data/etc/pki
 pki_registry_path=%(pki_root_prefix)s/etc/sysconfig/pki
 pki_instance_path=%(pki_path)s/%(pki_instance_name)s
 pki_instance_log_path=%(pki_log_path)s/%(pki_instance_name)s
--- /usr/share/authconfig/authinfo.py	2017-08-04 09:05:35.000000000 +0000
+++ /usr/share/authconfig/authinfo.py	2018-09-15 11:06:10.604367364 +0000
@@ -52,7 +52,7 @@
 except ImportError:
         SSSDConfig = None
 
-SYSCONFDIR = "/etc"
+SYSCONFDIR = "/data/etc"
 AUTH_PAM_SERVICE = "system-auth"
 AUTH_PAM_SERVICE_AC = "system-auth-ac"
 
--- /usr/lib/tmpfiles.d/var.conf	2016-10-01 15:43:35.000000000 +0000
+++ /usr/lib/tmpfiles.d/var.conf	2018-12-14 09:56:44.416153076 +0000
@@ -12,8 +12,8 @@
 L /var/run - - - - ../run
 
 d /var/log 0755 - - -
-f /var/log/wtmp 0664 root utmp -
-f /var/log/btmp 0600 root utmp -
+L /var/log/wtmp - - - - /data/var/log/wtmp
+L /var/log/btmp - - - - /data/var/log/btmp
 
 d /var/cache 0755 - - -
 
#
# Force writing journal to /run/log/journal which we symlink to
# /data/var/log/journal
#
--- /etc/systemd/journald.conf	2016-02-01 14:04:05.000000000 +0000
+++ /etc/systemd/journald.conf	2018-12-14 08:49:25.189295557 +0000
@@ -12,7 +12,7 @@
 # See journald.conf(5) for details.
 
 [Journal]
-#Storage=auto
+Storage=volatile
 #Compress=yes
 #Seal=yes
 #SplitMode=uid
